benchmarks/esbuild.js

```diff
@@ -19,4 +20,9 @@ const runBenchmarks = async function () {
   console.log(`${largeDepsEsbuild}ms`)
 }
 
-runBenchmarks()
+try {
+  await runBenchmarks()
```

Comment:
Just as information you could now use the `vitest bench` benchmarking functionality. Nothing to change here but just as a FYI

Reply:
Yeah, I was wondering, but didn't want to add even more changes.
Not super happy with vitest yet, had to do some workarounds, so do not want to buy in yet fully :)